pkg/test/resourcefixture/testdata/resourceoverrides/dnsrecordset/routingpolicieswrrhealthchecks/dependencies.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeNetwork
+metadata:
```

Comment:
Same comment for project-id annotation.

Reply:
I added all the project-id annotation back. They don't make a different in my local testing. And I haven't tested with the trick to copy the change to Gerrit. But I am OK with keeping the project-id annotation.

Reply:
Thank you!